onboarding.adoc

```diff
@@ -18,6 +18,75 @@ Ut imperdiet egestas diam nec pretium.
+Speeding up the acceptance process is critical to growing your contributor base.
+
+Fortunately, there are tools for that.
+
+=== Continuous integration, continuous delivery, and speed
+
+The primary methods that software projects—both community and corporate—have been using to speed up integration of patches is continuous integration and continuous delivery (CI/CD).
+The idea behind these technologies is to automate, as much as possible, the acceptance and integration of patches and new features.
+Continuous integration involves accepting pull requests and patches as soon as they pass automated tests and a simple evaluation threshold.
+Continuous delivery involves deploying new versions of the software whenever a new, tested build is ready.
+
+Much of the focus on CI/CD has been on speeding up delivery of software to users.
+But these technologies can also be very helpful for providing contributors with "immediate gratification" by seeing their patches get merged and go out quickly.
+This is perhaps their most valuable aspect, as the pace of contributions is often determined by the pace of acceptance.
+If you accept faster, then you usually get new code faster.
+
+However, implementing CI/CD can be difficult for project leaders, because their automated testing isn't comprehensive, their builds aren't fully automated, and the consequences of letting a major bug get merged into the code can be very severe.
+Despite a good deal of lip service to the values and desirability of having a continuous code pipeline, irregular batch committer review is generally the reality, especially for smaller projects.
+Additionally, CI/CD software like Gerrit, Jenkins, and Travis can be difficult to set up in a way that supports projects' real releases.
+
+There is, however, one area where CI/CD is easier to implement—and most projects should be doing it right now.
+
+=== Automating documentation contributions
+
+Documentation is one area in which most open source projects sorely need more contributions, and where acceptance of imperfect submissions carries relatively low risk.
+If your project is like most, you have a long list of things with obsolete or non-existent documentation, few documentation contributors, and little time for documentation review.
+And unless you have a large, mature, and heavily cross-referenced documentation set, the problems caused by accepting documentation errors are smaller than those caused by not having documentation on certain features at all.
+
+This means that, for most projects, self-serve CI/CD for documentation would be a real benefit.
+This is the reason why many projects adopt wikis for their documentation, since those allow for immediate editing and display.
+However, wikis make structuring documentation, rendering it in formats other than screen, or tying it to specific release versions difficult.
+For these reasons (and others), some projects have transitioned to building docs using CI/CD in DocBook, Sphinx, or AsciiDoc.
```

Comment:
Need to update this with the current list of popular documentation formats.

Reply:
Great! Thank you.